### PR TITLE
Add Locklizard Safeguard Viewer v2.5.81

### PR DESCRIPTION
**Pull request by chgallen**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/locklizard-safeguard-viewer.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'locklizard-safeguard-viewer' do
+  version ':latest'
```
Comment (miccal) [inline Casks/locklizard-safeguard-viewer.rb:2]: Please change this to:
```
version '2.5.81'
```

File: Casks/locklizard-safeguard-viewer.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 'd77a65e7f19aae939c5bbe2eb0db4ac7e678d6aa9dde6a9fdfd55964673bd08c'
+
+  url 'https://downloads.locklizard.com/PDCViewerSetupOSX.zip'
```
Comment (miccal) [inline Casks/locklizard-safeguard-viewer.rb:5]: I think the following may make a suitable `appcast`:
```
appcast 'https://www.locklizard.com/download_pdf_viewers/#mac-viewer'
```

Comment (chgallen) [reply Casks/locklizard-safeguard-viewer.rb:5]: Done ✅

File: Casks/locklizard-safeguard-viewer.rb (hunk 1 above)
Comment (chgallen) [reply Casks/locklizard-safeguard-viewer.rb:2]: Yeah, I wasn't sure whether to use latest or 2.5.81. There isn't a versioned url for the download, but the unzipped package contains the version number. Should I add 2.5.81 as the version and then use #{version.no_dots} in the pkg stanza (even though the url is generic)?

Comment (miccal) [reply Casks/locklizard-safeguard-viewer.rb:2]: Yes please.

Comment (chgallen) [reply Casks/locklizard-safeguard-viewer.rb]: Thanks! Done ✅

**Comment by vitorgalvao** [comment]
Welcome, and thank you for the contribution.

**Pull request merged**